src/Voron/Impl/Scratch/ScratchBufferFile.cs

```diff
@@ -43,6 +46,8 @@ public ScratchBufferFile(AbstractPager scratchPager, int scratchNumber)
             _scratchPager = scratchPager;
             _scratchNumber = scratchNumber;
             _allocatedPagesCount = 0;
+            _thisDebugInfo = (_scratchPager.DebugInfo, scratchNumber.ToString());
+            MemoryInformation.MemoryMappedAllocatedMemory[_thisDebugInfo] = _allocatedPagesCount * Constants.Storage.PageSize;
```

Comment:
why do we need this?
Also, there may be multiple concurrent maps to the same file